dashboard/src/components/PackageListSelectFilters.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import { usePackageStore } from "@/stores/package";
+import { PackageListStatusEnum } from "@/openapi-generator";
+
+const packageStore = usePackageStore();
+</script>
+
+<template>
+  <div class="d-flex flex-wrap gap-3 mb-3 p-3 border bg-light">
+    <h3 class="mb-0 pe-3 border-end">Filters</h3>
+    <div class="d-flex gap-2">
+      <label for="filter-status" class="align-self-center">Status</label>
+      <select
+        id="filter-status"
+        v-model="packageStore.filters.status"
+        @change="packageStore.fetchPackages(1)"
+        class="form-select"
+      >
+        <option value="">any</option>
+        <option v-for="item of PackageListStatusEnum" :value="item">
+          {{ item }}
+        </option>
+      </select>
+    </div>
+  </div>
+</template>
```

Comment:
We can remove this component now, right?

Reply:
From @fiver-watson's comments I thought we might do so A / B testing of the dropdown vs tab status filters, so I kept this component for that.

Reply:
Okay, it will be nice to add some tests if we keep it. Doesn't need to happen in this PR.

Reply:
And make it a real component ;)